app/directives/megasearch.js

```diff
@@ -11,6 +11,14 @@ angular
 			},
 			link: function(scope, elem) {
 				scope.provideSuggestions = function(keyCode) {
+
+					// size the mega search box to match the input field
+					if($location.path()==='/'){
+						$('#search-terms #megasearchbox').width($('#search-terms #searchbox').width()-18);
```

Comment:
I am not talking about `$location`, which is part of angular, but the selectors like `$('#search-terms #megasearchbox')` which are jQuery and not angular syntax.